elifecrossref/dates.py

```diff
@@ -16,3 +37,8 @@ def set_date_detail(parent, pub_date):
     day_tag.text = str(pub_date.tm_mday).zfill(2)
     year_tag = SubElement(parent, "year")
     year_tag.text = str(pub_date.tm_year)
+
+
+def iso_date_string(pub_date):
+    """format a date object into an iso format date string e.g. 2019-12-31"""
+    return time.strftime('%Y-%m-%d', pub_date) if pub_date else None
```

Comment:
> format a date object

I'm not sure what type the `pub_date` value is, but it isn't a *date object*:

```
$ python
Python 3.8.2 (default, Apr  8 2020, 14:31:25) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from datetime import date, datetime, time
>>> time.strftime("%Y-%m-%d", datetime.now())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: descriptor 'strftime' for 'datetime.time' objects doesn't apply to a 'str' object
>>> time.strftime("%Y-%m-%d", date.today())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: descriptor 'strftime' for 'datetime.time' objects doesn't apply to a 'str' object
```

fix docstr

Reply:
turns out `time` and `datetime.time` are two different things but the docstr still needs changing

Reply:
Yup, thanks, it was not an accurate description.

Reply:
Input supplied is a `time.struct_time` object.